src/main/org/apache/tools/ant/AntClassLoader.java

```diff
@@ -297,6 +298,7 @@ public AntClassLoader() {
      *                elements are set up to start with.
      */
     public AntClassLoader(final Project project, final Path classpath) {
+        super(parent);
```

Comment:
I think to make it clear what's being done here, it's better to do `super(null)` instead of using `parent` which is `null`.